Source/Relativity.Test.Helpers/Relativity.Test.Helpers/Mail/MailTrapInboxModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Relativity.Test.Helpers.Mail
+{
+	public class MailTrapInboxModel : IMailInboxModel
+	{
+		public int Id { get; set; }
+		public int company_id { get; set; }

```

Comment:
It may be better to use the .net convention of CompanyId instead. and use [JsonProperties](https://www.newtonsoft.com/json/help/html/JsonPropertyName.htm) instead.